model/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//model:src/main/proto/preprocess_protos.bzl", "process")
+
+proto_library(
+    name = "proto_lib",
+    srcs = ["src/main/proto/event_logger.proto",
+            "src/main/proto/example.proto",
+            "src/main/proto/interaction_object.proto",
+            "src/main/proto/onboarding.proto",
+            "src/main/proto/profile.proto",
+            "src/main/proto/subtitled_html.proto",
+            "src/main/proto/thumbnail.proto",
+            "src/main/proto/translation.proto",
+            "src/main/proto/voiceover.proto",
+    ],
+    data = [
+            ":process_exploration",
```

Comment:
For consistency, we should use four spaces tab continuity. Ditto elsewhere.

Reply:
Okay, will change! Thanks!

Reply:
Just realised that for `.bzl` files, I think Bazel expects 2 space tabs. For native rule headers like `native.genrule()`, it actually throws an indentation error if I do 4 space tabs. It doesn't mind having 4 space tabs for the attributes inside the rule though - should I change those indentations or leave them?

Reply:
Ah, in that case follow whatever Bazel expects & ensure that tab continuity is consistent.

Reply:
Okay!